apps/sms/services/README.md

```diff
@@ -0,0 +1,91 @@
+# The layers in the Messages application
+
+## Terms explanation
+
+There are 3 layers whose interfaces managed by the Bridge (aka [threads.js](https://github.com/gaia-components/threads/)):
+
+* Services define high-level semantics to access the model. They are the only
```

Comment:
> Services (as in Service Layer) are already "real" services (as in Bridge-managed services -- we need a better terminology :p) ,and will use bridge.service. They may use bridge.client to call other services/managers/wrappers.

Mmm so if ```JService``` (Julien's Service :D) creates internally ```BService``` (Bridge Service aka ```bridge.service```) and as far as I understood it's hosted in a ```ShW``` (SharedWorker), ```DW``` (DedicatedWorker) or ```SW``` (ServiceWorker), how it can ```be directly called by the views``` without ```bridge.client``` or ```OClient``` (Oleg's Client aka wrapper around ```bridge.client```)? 

> What I'm not sure yet is whether the "Manager" layer should be bridge-managed services or merely JS libs that we can import into Service layer services. I'd rather take the simpler road for a start (so "Managers" are reusable libs). Still the distinction is important to me, so that we don't duplicate code and think in reusable components rather than organically refactor things because technically it's better.

So is ```Manager``` just a lib around some of the functionality, like ```DBManager```, ```NotificationManager``` or more high level like ```ConversationManager``` that doesn't know anything aboout ```bridge``` and just helps with some stuff (e.g. wraps IDB work)?

And ```Wrapper``` is just a mix of ```Manager``` (wraps API work) and ```Service``` (uses ```bridge``` internally because it's hosted in iframe/window and can't be accessed directly from worker)?